packages/snap-preact/src/Instantiators/RecommendationInstantiator.tsx

```diff
@@ -192,7 +192,10 @@ export class RecommendationInstantiator {
 				const defaultGlobals = {
 					limits: 20,
 				};
-				const globals = deepmerge(deepmerge(defaultGlobals, this.config.client?.globals || {}), contextGlobals);
+				const globals = deepmerge(
+					deepmerge(deepmerge(defaultGlobals, this.config.client?.globals || {}), contextGlobals),
+					(this.config.config?.globals as any) || {}
```

Comment:
This looks related to how the `options.filters` was incorrectly added in the controller code, instead the "contextGlobals" should be updated above, similarly to how the `options.categories` is done on line 166 above.

Reply:
This was discussed and decided an acceptable bugfix